include/eblob/blob.h

```diff
@@ -879,6 +885,7 @@ static inline const char *eblob_dump_blob_flags(unsigned int flags) {
 		{ EBLOB_DISABLE_THREADS,		"disabled_threads"},
 		{ EBLOB_AUTO_INDEXSORT,			"auto_indexsort"},
 		{ EBLOB_USE_VIEWS,			"use_views"},
+		{ EBLOB_SORT_BY_POS,			"sort_by_pos"},
```

Comment:
is buffer still big enough to hold all possible flags?

Reply:
length of all flags is 178 but I don't know how much we need at eblob_dump_flags_raw.

Reply:
Let's increase it to 512 just in case

Reply:
done